docs/advanced-topics/webhook.asciidoc

```diff
@@ -0,0 +1,19 @@
+:page_id: webhook
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Webhook namespace selectors 
+
+Users are now allowed to set `namespaceSelector` and `objectSelector` on the webhook. The webhook name is generated as ``<operator-name>.<operator-namespace>.k8s.elastic.co` so that multiple operators can be installed side-by-side in the same cluster. 
```

Comment:
Users can only do this when installing the operator via the Helm chart. We should make that clear.